go/cli/mcap/cmd/cat.go

```diff
@@ -304,6 +318,13 @@ var catCmd = &cobra.Command{
 		readingStdin := (stat.Mode()&os.ModeCharDevice == 0 && len(args) == 0)
 		// stdin is a special case, since we can't seek
 
+		if catStartSec > 0 && catStartNano > 0 {
```

Comment:
nit: is there any mechanism to see which flags were actually passed? If `--start-nsec 0` is valid as "since epoch", then we might also want to prevent `--start-sec 123 --start-nsec 0` since I might otherwise expect the last flag to be used?

Reply:
It turns out that we can do even better: use Cobra's built-in `MarkFlagsMutuallyExclusive` for this very job. Thanks for the hint!